docs/getting-started/walk-through/advanced-hera-features.md

```diff
@@ -89,12 +89,35 @@ conflicts.
 ## Experimental Features
 
 From time to time, Hera will release a new feature under the "experimental feature" flag while we develop the feature
-and ensure stability. Currently this is used for the `RunnerScriptConstructor` seen in the
-[runner script example](../../examples/workflows/callable_script.md).
+and ensure stability. 
 
 To enable experimental features you must set the feature by name to `True` in the `global_config.experimental_features`
 dictionary before using the feature:
 
+```py
+global_config.experimental_features["NAME_OF_FEATURE"] = True
+```
+
+### Currently supported experimental features:
+
+#### `RunnerScriptConstructor`
+The `RunnerScriptConstructor` found in `hera.workflows.script` and seen in the
+[callable script example](../../examples/workflows/callable_script.md) is a robust way to run Python functions on Argo.
+The image used by the script should be built from the source code package itself and its dependencies, so that the
+source code's functions, dependencies, and Hera itself are available to run. The `RunnerScriptConstructor` is also
+compatible with Pydantic so supports deserializing inputs to Python objects and serializing outputs to json strings. It
+must be enabled with the `script_runner` feature flag as below.
+
 ```py
 global_config.experimental_features["script_runner"] = True
 ```
+
+
+#### Script Annotations
+An annotation based system for simplifying script parameter creation, as seen in the [script annotations example](../../examples/workflows/script_annotations_combined_new.md)
```

Comment:
Would be good to demonstrate the name aliasing aspect and expand this bit of documentation as well (as it's the only docs we have for this). Feel free to expand and sell it a bit more!

```suggestion
An annotation based system for simplifying script parameter creation, as seen in the [script annotations example](../../examples/workflows/script_annotations_combined_new.md).

Script annotations can work on top of the `RunnerScriptConstructor` for name aliasing of function
parameters, in particular to ensure a public `kebab-case` parameter, while using a `snake_case`
Python function parameter. When using a `RunnerScriptConstructor`, an environment variable
`hera__script_annotations` will be added to the Script template.

Script annotations also work with the regular `InlineScriptConstructor` for
generating valid template parameters in the yaml instead of using the `@script` decorator.
```

Maybe add a Python snippet showing old vs new?